client/src/components/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+
+const Button = styled.button`
+  height: 45px;
+  width: 130px;
+  border-radius: 25px;
+  background: var(--main-color);
+  border: none;
+  cursor: pointer;
+  color: var(--button-color);
+  font-weight: bold;
+`;
+
+export const Buttons = ({ title }) => {
+  return <Button>{title}</Button>;
+};
+
+Buttons.propTypes = {
+  title: PropTypes.string.isRequired,
+};
```

Comment:
Why is `Buttons` rendering a single `Button`? 
I guess you wanted to avoid a naming conflict here.

styled components are React components too, so you could directly export `Button` here.

Wdyt about this?

```suggestion
import styled from "styled-components/macro";

export const Button = styled.button`
  height: 45px;
  width: 130px;
  border-radius: 25px;
  background: var(--main-color);
  border: none;
  cursor: pointer;
  color: var(--button-color);
  font-weight: bold;
`;
```

You have to change the usage to from `<Button title="test" />` to `<Button>test</Button>`

Reply:
Thanks for the tip!